spec/features/exhibit_index_spec.rb

```diff
@@ -1,7 +1,42 @@
 require 'rails_helper'
 
 describe 'the exhibit home page' do
-  it 'loads the home page' do
+  before do
+    login_as(user, scope: :user)
     visit '/'
   end
+
+  describe 'user pull-down menu' do
+    context 'with a non-admin user' do
+      let(:user) { FactoryGirl.create(:curator) }
+      it 'hides "create exhibit" link' do
+        # This fails because user is an admin (inexplicably)
```

Comment:
When I drop in through `byebug` and call `user.superadmin?`, I get `true`. I get the same with a totally vanilla FG user as well.